docs/run-component-guide.md

```diff
@@ -1,24 +1,24 @@
-## Running the gem as a standalone app
+## Running the component guide as a standalone app
 
 If you want to work on this gem directly, without having to add it to another
-application, you can run it in the development vm or on your local machine with:
+application, you can run it on your local machine or using [govuk-docker](https://github.com/alphagov/govuk-docker). You'll need [Ruby](https://www.ruby-lang.org/en/downloads/) and [Node.js](https://nodejs.org/en/) installed to get this project running.
 
+### Running the component guide on your local machine
 ```
+cd ~/govuk/govuk_publishing_components
 ./startup.sh
 ```
 
-The app should start on http://localhost:3212 or http://publishing-components.dev.gov.uk/component-guide on GOV.UK development machines.
+The app should start on [http://localhost:3212/component-guide/](http://localhost:3212/component-guide/).
 
-
-#### Running with bowl on the VM
+### Running the component guide in [govuk-docker](https://github.com/alphagov/govuk-docker)
 
 ```
-$ cd /var/govuk/govuk_publishing_components
-$ bundle
-$ npm install
+cd ~/govuk/govuk-docker
+make govuk_publishing_components
 
-$ cd /var/govuk/govuk-puppet/development-vm
-$ bowl publishing-components
+cd ~/govuk/govuk_publishing_components
+govuk-docker up govuk_publishing_components-app
```

Comment:
If you `cd` into the folder you can just run `govuk-docker-up`; the command `govuk-docker up govuk_publishing_components-app` can be run from any folder.

```suggestion
govuk-docker-up
```